quic/transport/ngtcp2/native/connection.nim

```diff
@@ -131,10 +132,11 @@ proc tryReceive(connection: Ngtcp2Connection, datagram: openArray[byte],
     raise newException(Ngtcp2ConnectionClosed, "connection no longer exists")
 
   var packetInfo: ngtcp2_pkt_info
-  packetInfo.ecn = ecn.uint32
-  checkResult ngtcp2_conn_read_pkt(
+  packetInfo.ecn = ecn.uint8
+  checkResult ngtcp2_conn_read_pkt_versioned(
     conn,
     connection.path.toPathPtr,
+    NGTCP2_PKT_INFO_V1,
     unsafeAddr packetInfo,
```

Comment:
I saw a lot of `unsafeAddr` in this code and from what I understand `unsafeAddr` is an alternative to `addr` when used on a `let` variable. For this one, `unsafeAddr` looks overkill.
It's not really the point of this PR though, but it may be interesting to check this case by case in another PR.

Reply:
done